### DISCUSSION TITLE
for loop

### DISCUSSION THREAD

**Discussion by Decodetalkers** [slint-ui/slint#1354] [Q&A] [answered]
Can slint ui do such thing like
```
for i in 0..10 : Rectangle{}
```

**Comment by ogoffart** [comment, marked as answer]
Yes:


```
for i in 10: Rectangle {}
```

**Reply by Decodetalkers** [reply]
Another question. How can I set a button unclickedable

**Reply by ogoffart** [reply]
you can use `enabled:false`

**Reply by Decodetalkers** [reply]
> you can use `enabled:false`

Thanks , and does slint support enum?

**Reply by tronical** [reply]
Unfortunately enums are not supported yet. See also https://github.com/slint-ui/slint/issues/150
